index.html

```diff
@@ -0,0 +1,10 @@
+<html>
+    <header>
+        <title>Microverse Project</title>
+        <link rel="stylesheet" href="style.css">
+    </header>
+
+    <body>
+        <h1>Hello Microverse!</h1>
+    </body>
```

Comment:
- [ ] Please as part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO